pkg/scheduler/scheduler.go

```diff
@@ -55,12 +69,56 @@ const (
 // Scheduler watches for new unscheduled pods. It attempts to find
 // nodes that they fit on and writes bindings back to the api server.
 type Scheduler struct {
```

Comment:
I'd like to find out which of these fields can be made lower case without breaking anything. I'm OK with making that change in a follow up PR.